src/App.tsx

```diff
@@ -419,6 +424,20 @@ export class App extends React.PureComponent <unknown, AppState> {
 				<ThemeProvider theme={theme}>
 					<Container maxWidth='xl'>
 						<Box className='row' sx={{ bgcolor: '#ffffff80', minHeight: '100vh' }}>
+							{this.state.currentPage == Pages.yearRecap || this.state.showDashboard ?
```

Comment:
Only use one ternary here and wrap your true case (the typography and button) in a fragment.